internal/staging/stdscript/scriptv0_test.go

```diff
@@ -1235,6 +1259,85 @@ func TestExtractTreasuryGenScriptHashV0(t *testing.T) {
 	}
 }
 
+// TestProvablyPruneableScriptV0 ensures generating a version 0
+// provably-pruneable nulldata script works as intended for
```

Comment:
s / ... works as intended for / ... works as intend.

Reply:
Updated.